packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -3004,7 +3008,7 @@ Generated by [AVA](https://ava.li).
       saveConfig: true␊
       testOpts:␊
         pluginsListUrl: test␊
-        silentLingeringProcesses: true␊
+        silentLingeringProcesses: 'true'␊
```

Comment:
Is this change due to `useBinary: true`?

Reply:
Yes :)